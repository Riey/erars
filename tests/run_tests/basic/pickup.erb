@SYSTEM_TITLE
ADDCHARA 3
ADDCHARA 3
ADDCHARA 3
ADDCHARA 3
ADDCHARA 3

TARGET = 3
NAME = TEST

PRINTFORML TARGET = {TARGET} NAME:TARGET = %NAME:TARGET%

PICKUPCHARA 2, 3

PRINTFORML TARGET = {TARGET} NAME:TARGET = %NAME:1%
