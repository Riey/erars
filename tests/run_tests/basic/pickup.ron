[
    Print("TARGET = 3 NAME:TARGET = TEST"), NewLine,
    Print("TARGET = 1 NAME:TARGET = TEST"), NewLine,
]