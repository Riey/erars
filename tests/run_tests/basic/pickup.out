TARGET = 3 NAME:TARGET = TEST
TARGET = 1 NAME:TARGET = TEST
